app/Filament/Resources/OrganisationResource/RelationManagers/DocumentsRelationManager.php

```diff
@@ -108,7 +128,14 @@ public static function table(Table $table): Table
                 TextColumn::make('expires_at')
                     ->label(__('document.field.expires_at'))
                     ->date()
-                    ->sortable(),
+                    ->sortable()
+                    ->formatStateUsing(function (Document $record) {
+                        if (DocumentType::protocol->isNot($record->type)) {
+                            return $record->expires_at?->format(config('tables.date_format'));
+                        }
+
+                        return $record->expires_at?->format(config('tables.date_format')) ?: __('document.field.never_expires');
+                    }),
```

Comment:
```suggestion
                    ->formatStateUsing(function (Document $record, $state) {
                        $fallback = DocumentType::protocol->is($record->type)
                            ? __('document.field.never_expires')
                            : null;

                        return $state ?? $fallback;
                    }),
```

Hey, a couple of things here:

1. `$state` is available on component callbacks and it contains the value of the model attribute identified in the component name. In this case, `$state` will already contain the value of `$record->expires_at`;
2. You don't heed to use `->format()` here. It's enough to return a Carbon instance, since it's already being covered by `->date()` above;
3. If specifically checking for null values, it's clearer  to use the null-coalescing operator `??` instead of the short ternary `?:`;
4. Reformatted for legibility;
5. This should also be applied to the `expires_at` TextColumn in the DocumentResource class.